templates/partners/silicon/index.html

```diff
@@ -22,350 +25,478 @@
   ] %}
   {% include '/partial/_breadcrumbs.html' %}
```

Comment:
The design doesn't have this breadcrumb and I couldn't see anything in the notes about it, but I see this has had a design review so maybe it's fine to leave it there...